.env.sample

```
@@ -3,6 +3,7 @@ NODE_ENV=development # or production
 # AI Keys
 DEEPSEEK_API_KEY=
 EMBEDDING_API_KEY=
+OPENAI_API_KEY=  # Required for text-to-speech audio generation
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)